client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	Sport: i18n.translate( 'Sport' ),
+	Sports: i18n.translate( 'Sports' ),
+	'Stationary supplies': i18n.translate( 'Stationary supplies' ),
+	Statistics: i18n.translate( 'Statistics' ),
+	'Steel protection': i18n.translate( 'Steel protection' ),
+	'Stock market': i18n.translate( 'Stock market' ),
+	Stocks: i18n.translate( 'Stocks' ),
+	Store: i18n.translate( 'Store' ),
+	Storefront: i18n.translate( 'Storefront' ),
+	Stories: i18n.translate( 'Stories' ),
+	Student: i18n.translate( 'Student' ),
+	'Student association': i18n.translate( 'Student association' ),
+	'Student housing': i18n.translate( 'Student housing' ),
+	'Students union': i18n.translate( 'Students union' ),
+	Style: i18n.translate( 'Style' ),
+	Stylist: i18n.translate( 'Stylist' ),
+	Subscriptions: i18n.translate( 'Subscriptions' ),
+	'Suicide prevention': i18n.translate( 'Suicide prevention' ),
+	Supplements: i18n.translate( 'Supplements' ),
+	Supplies: i18n.translate( 'Supplies' ),
+	Support: i18n.translate( 'Support' ),
+	'Sustainable development': i18n.translate( 'Sustainable development' ),
+	'Swimmping pools': i18n.translate( 'Swimmping pools' ),
+	'Swimmping pools repairs': i18n.translate( 'Swimmping pools repairs' ),
+	TV: i18n.translate( 'TV' ),
+	'TV shows': i18n.translate( 'TV shows' ),
+	'Tactical gear': i18n.translate( 'Tactical gear' ),
+	'Tattoo parlor': i18n.translate( 'Tattoo parlor' ),
+	Tattoos: i18n.translate( 'Tattoos' ),
+	Taxes: i18n.translate( 'Taxes' ),
+	'Taxi business': i18n.translate( 'Taxi business' ),
+	Taxidermy: i18n.translate( 'Taxidermy' ),
+	Teaching: i18n.translate( 'Teaching' ),
+	Tech: i18n.translate( 'Tech' ),
+	'Tech blog': i18n.translate( 'Tech blog' ),
+	'Technical recruiting': i18n.translate( 'Technical recruiting' ),
+	'Technical support': i18n.translate( 'Technical support' ),
+	Technology: i18n.translate( 'Technology' ),
+	Telecommunications: i18n.translate( 'Telecommunications' ),
+	Therapy: i18n.translate( 'Therapy' ),
+	Thoughts: i18n.translate( 'Thoughts' ),
+	Time: i18n.translate( 'Time' ),
+	Tips: i18n.translate( 'Tips' ),
+	Tourism: i18n.translate( 'Tourism' ),
+	'Tourism accomodation': i18n.translate( 'Tourism accomodation' ),
+	Toys: i18n.translate( 'Toys' ),
+	Trading: i18n.translate( 'Trading' ),
+	'Translation services': i18n.translate( 'Translation services' ),
+	Transport: i18n.translate( 'Transport' ),
+	Travel: i18n.translate( 'Travel' ),
+	'Travel agent': i18n.translate( 'Travel agent' ),
+	'Traveller blog': i18n.translate( 'Traveller blog' ),
+	Troubleshooting: i18n.translate( 'Troubleshooting' ),
+	Turorials: i18n.translate( 'Turorials' ),
+	UI: i18n.translate( 'UI' ),
+	'UI designer': i18n.translate( 'UI designer' ),
+	University: i18n.translate( 'University' ),
+	'User interface': i18n.translate( 'User interface' ),
+	'User interface design': i18n.translate( 'User interface design' ),
+	VR: i18n.translate( 'VR' ),
+	'VR glasses': i18n.translate( 'VR glasses' ),
+	'Vacation rental': i18n.translate( 'Vacation rental' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Hotels & Vacation Rentals' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
👎